src/tox/session/commands/run/parallel.py

```diff
@@ -13,6 +13,7 @@
 def run_parallel(config, venv_dict):
     """here we'll just start parallel sub-processes"""
     live_out = config.option.parallel_live
+    nospinner = config.option.parallel_nospinner
```

Comment:
On the second thought. It's probably better to use less negation in vars. So this one would be less confusing:
```suggestion
    spinner = not config.option.parallel_nospinner
```